frontend/src/components/dashboard/OurMission.js

```diff
@@ -3,44 +3,102 @@ import {makeStyles} from '@material-ui/core/styles';
+        container
+        direction="row"
+        alignItems="center"
+        justify="center"
+        spacing={4}
+      >
+        <Grid
+          container
+          item
+          xs={12}
+          md={5}
+          className={classes.title}
+          alignItems="center"
+          justify="center"
+          direction="column"
+          spacing={1}
+        >
+          <Grid item>
+            <Typography variant="h2" align="center">
+              Our Mission
+            </Typography>
+          </Grid>
+          <Grid item>
+            <img src={LogoImg} alt="logo" className={classes.image}/>
+          </Grid>
         </Grid>
-        <Grid item xs={12} md={8}>
-          <Typography variant="body1">
-            With the goal of making art universally accessible,
-            especially to those that are visually-impaired,
-            we created a website that provides a virtual
-            art experience.  This website allows artists
-            and art-lovers of all abilities to both enjoy art
-            and share their own artwork.  The website is
-            screen-reader compatible and keyboard navigable.
-            Every piece of art displayed on the website has
-            a corresponding description that expands beyond
-            a one line ALT text, and this description will
-            be read aloud when an artwork is selected.
-            While much of the artwork
-            displayed on the website is drawn from an
-            art museum’s public API, we also want to provide a
-            platform for any artist to share their art.
-            Thus, we include an accessible form for
-            users to upload and write descriptions of
-            their own artwork.
-          </Typography>
+        <Grid item xs={12} md={7}>
+          <List>
+            <ListItem>
+              <ListItemAvatar>
+                <Avatar className={classes.avatar}>
+                  <UniversalIcon fontSize="large"/>
+                </Avatar>
+              </ListItemAvatar>
+              <ListItemText
+                primary="Make art universally accessible."
+                primaryTypographyProps={primaryTypographyProps}
+              />
+            </ListItem>
+            <ListItem>
+              <ListItemAvatar>
+                <Avatar className={classes.avatar}>
+                  <AccessibilityIcon fontSize="large"/>
+                </Avatar>
+              </ListItemAvatar>
+              <ListItemText
+                primary="Made with accessibility in mind first."
+                primaryTypographyProps={primaryTypographyProps}
+              />
+            </ListItem>
+            <ListItem>
+              <ListItemAvatar>
+                <Avatar className={classes.avatar}>
+                  <ArtIcon fontSize="large"/>
+                </Avatar>
+              </ListItemAvatar>
+              <ListItemText
+                primary="Showcase art from lesser known artists."
```

Comment:
I feel like this could imply that we are curating artworks from lesser known artists ourselves (like we search for and find lesser known artists and put their art on the website ourselves as opposed to users uploading themselves).  Maybe something like "Showcase your own artwork" ?